recipes/ngsutils/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: ngsutils
+  version: '0.5.9'
+
+source:
+  url: https://github.com/ngsutils/ngsutils/archive/ngsutils-0.5.9.tar.gz
+  md5: 6212d6de740c42aa2c6b5f6785465860
+
+build:
+  number: 0
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python >=2.7.0
+    - pysam
+    - cython
+    - samtools
+    - eta
+    - coverage
+    - swalign
+  run:
+    - python >=2.7.0
+    - pysam
+    - cython
+    - samtools
+    - eta
+    - coverage
+    - swalign
+
+test:
+  commands:
+    - python -m unittest discover ngsutils
+
+about:
+  home: http://ngsutils.org
+  license: LICENSE
```

Comment:
Many thanks for your improvements. 

I'm confused by this one, does the 'license:' section of the meta.yaml require a link to the license file included within the source directory or just the license name (i.e. GPL, BSD, etc.)? The "LICENSE" I've put here is supposed to be referring to the file in the source directory.